_tutorials/2016-05-10-iosxr-ansible.md

```diff
@@ -40,88 +52,79 @@ Vagrant.configure(2) do |config|
   config.vm.define "ubuntu" do |ubuntu|
     ubuntu.vm.box = "ubuntu/trusty64"
     ubuntu.vm.network :private_network, virtualbox__intnet: "link1", ip: "10.1.1.10"
+    ubuntu.vm.provision :shell, path: "ubuntu.sh", privileged: false
+    ssh_pub_key = File.readlines("#{Dir.home}/.ssh/id_rsa.pub").first.strip
+    #ubuntu.vm.provision "shell", inline: "echo #{ssh_pub_key}"
   end
 
   config.vm.define "xr" do |xr|
     xr.vm.box = "xrv64"
+    #xr.vm.provision 'shell', inline: "echo #{ssh_pub_key} >> /home/vagrant/.ssh/authorized_keys", privileged: false
     xr.vm.network :private_network, virtualbox__intnet: "link1", ip: "10.1.1.20"
   end
-
+   
 end
 
 ```
 
 
-Next step to create Vagrantfile and boot up boxes:
+Copy files from [Github](https://github.com/Maikor/IOSXR-Ansible-tutorial), change dir to this folder and you are ready to boot up boxes:
 
+```
 mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
-$ vagrant up
+$ ls
+ubuntu.sh*  Vagrantfile  xr-config
 
+mkorshun@MKORSHUN-2JPYH MINGW64 ~/Documents/workCisco/tutorial
+$ vagrant up
+```
 
 
 ### Ubuntu box pre-configuration
```

Comment:
The entire ubuntu box pre-configuration section, should be inside  a {: .notice--info} class. These commands are not supposed to be executed by the user, so don't use code blocks to explain it.